packages/camera/camera_android_camerax/lib/src/image_capture.dart

```diff
@@ -0,0 +1,130 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart' show BinaryMessenger;
+
+import 'camerax_library.g.dart';
+import 'instance_manager.dart';
+import 'java_object.dart';
+import 'use_case.dart';
+
+/// Use case for picture taking.
+///
+/// See https://developer.android.com/reference/androidx/camera/core/ImageCapture.
+class ImageCapture extends UseCase {
+  /// Creates a [ImageCapture].
+  ImageCapture(
+      {BinaryMessenger? binaryMessenger,
+      InstanceManager? instanceManager,
+      this.targetFlashMode,
+      this.targetResolution})
```

Comment:
nit: you can add a comma at the end of targetResolution for a cleaner format. Same for the detached constructor